src/components/DashboardModule/dashboardmodule.scss

```diff
@@ -65,4 +64,23 @@
         font-size: 14px;
         display: block;
     }
-}
\ No newline at end of file
+}
+
+@media screen and (max-width:600px){
+    .dashboard__modules_container {
+        display:block !important;
+        width: 100%;
```

Comment:
"Cardinal sin of CSS"

Is there an alternative way to accomplish this that avoids `!important`?

Reply:
> "Cardinal sin of CSS"
Well played, sir. I shouldn't have pull requested before cleanup. Took me a while to find the right hook and before that I was playing quick and loose with the overrides to save time.
I'm sure I can fix that.

Reply:
Unfortunately, there's a lot of custom SCSS that's overriding the thoughtful presets that bulma provides and making it difficult to know where exactly to find the offending styles that are screwing things up and which I would need to override with even greater css specificity. In other words, this is one of those situations where the ball was already all tangled up before I got here and the only way to avoid unraveling it is !important.